src/main/java/Storage.java

```diff
@@ -0,0 +1,83 @@
+import java.awt.desktop.SystemEventListener;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Scanner;
+
+public class Storage {
+
+    String filePath = "data/duke.txt";
```

Comment:
I'm not sure if this path worked for the ```.jar``` file, but for mine, it led to an error since that directory doesn't exist. I'm not sure on this though.